go/vt/vtgate/gorpcvtgateconn/conn_test.go

```diff
@@ -40,16 +43,67 @@ func initEnv() {
 			vttest.LocalTeardown()
 			panic(err)
 		}
+		fmt.Println("Launched local cluster")
 	})
 }
 
-/*
 func TestMain(m *testing.M) {
 	r := m.Run()
 	vttest.LocalTeardown()
 	os.Exit(r)
 }
-*/
+
+func TestExecuteShard(t *testing.T) {
+	t.Skip("Take too much time to run, skipping...")
```

Comment:
It is intentional. Tests in conn_test.go take too much time to run, they need to launch a local vitess cluster and run several queries against it. It typically takes 20 - 30 secs for each test, so I comment it out. BTW, other tests in this file have been skipped already.